### PR TITLE
Fix: View TextFacsimileSplitView: add openseadragon as alternative viewer

### LINKED ISSUE #370: TextFacsimileSplitView: TEI Facsimile not loading
The TEI facsimile view was fixed at #356 #339
Maybe a side-effect

### PR DESCRIPTION
**Pull request by bwbohl**
closes #370 

Please, also see https://github.com/Edirom/Edirom-Online/issues/370#issuecomment-2206307716
for hints on how your data has to be encoded.

### CODE REVIEW

**Comment by riedde** [comment]
I'll check the status of this issue also comparing to #370 because I think there is potential for synergies.

File: app/view/window/text/TextFacsimileSplitView.js [hunk 1]
```diff
@@ -45,8 +45,11 @@ Ext.define('EdiromOnline.view.window.text.TextFacsimileSplitView', {
     	
     	if(me.image_server === 'leaflet'){
     		me.imageViewer = Ext.create('EdiromOnline.view.window.image.LeafletFacsimile');
-    	}
-    	else{
+    	}else if(image_server === "openseadragon") {
```
Comment (peterstadler) [inline app/view/window/text/TextFacsimileSplitView.js:48]:
```suggestion
    	}else if(me.image_server === "openseadragon") {
```

**Review by peterstadler — CHANGES_REQUESTED** [review]
Hi @bwbohl, @obertsalome and I were testing your PR and after fixing the variable reference `image_server` (see suggestion) it indeed remedies the issue #370 for @obertsalome 's edition of the Pintos! Many thanks!!

**Review by peterstadler — DISMISSED** [review]

File: app/view/window/text/TextFacsimileSplitView.js (hunk 1 above)
Comment (bwbohl) [reply app/view/window/text/TextFacsimileSplitView.js]: Thanks for spotting & fixing; I'm wondering why it worked with my test data… maybe?

Nevermind

**Review by peterstadler — APPROVED** [review]
works for us :)

**Pull request merged**